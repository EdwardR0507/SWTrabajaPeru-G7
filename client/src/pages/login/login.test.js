/*Importación de librerias y paquetes necesarios*/
import {SignIn, NavBar} from './index';
import React from 'react';
import { shallow } from 'enzyme';
/*Renderizado de la vista Login*/
describe("<SignIn></SignIn>", () => {
  it('render "Registrarse"', () => {
    const wrapper = shallow(<SignIn />);
    expect(wrapper).toMatchSnapshot()
  });
  it('should have the `NavBar` in "IniciarSesión"', () => {
    const wrapper = shallow(<SignIn />);
    expect(wrapper.containsMatchingElement(<NavBar></NavBar>)).toBe(false);
  });
  
});