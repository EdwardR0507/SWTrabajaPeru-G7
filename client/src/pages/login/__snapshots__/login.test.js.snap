// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SignIn></SignIn> render "Registrarse" 1`] = `ShallowWrapper {}`;
